src/script/descriptor.cpp

```diff
@@ -740,6 +755,17 @@ class PKDescriptor final : public DescriptorImpl
 public:
     PKDescriptor(std::unique_ptr<PubkeyProvider> prov, bool xonly = false) : DescriptorImpl(Vector(std::move(prov)), "pk"), m_xonly(xonly) {}
     bool IsSingleType() const final { return true; }
+
+    std::optional<int64_t> ScriptSize() const override { return 1 + (m_xonly ? 33 : 32) + 1; }
```

Comment:
In c8b0aeb534edce08366176275d5577c1e309eb6f "descriptor: introduce a method to get the satisfaction size"

This is backwards, should be 32 for `m_xonly`.

```suggestion
    std::optional<int64_t> ScriptSize() const override { return 1 + (m_xonly ? 32 : 33) + 1; }
```

Reply:
Woops. Thanks.